packages/node_modules/pouchdb-adapter-idb/src/index.js

```diff
@@ -379,6 +379,10 @@ function init(api, opts, callback) {
     var type = attachment.content_type;
 
     txn.objectStore(ATTACH_STORE).get(digest).onsuccess = function (e) {
+      if (!e.target.result) {
+        callback(null, null);
+        return;
+      }
```

Comment:
Will expand more outside a line comment, but I think we need pouch to know that we have a stub without an attachment and to expect that + not try to fetch the attachment, otherwise I think this could hide errors